tests/end-to-end/director/application_api_test.go

```diff
@@ -1273,15 +1276,15 @@ func fixSampleApplicationUpdateInput(placeholder string) graphql.ApplicationUpda
 	return graphql.ApplicationUpdateInput{
 		Name:           placeholder,
 		Description:    &placeholder,
-		HealthCheckURL: &placeholder,
+		HealthCheckURL: ptr.String("https://kyma-project.io"),
 	}
 }
 
 func fixSampleApplicationUpdateInputWithIntegrationSystem(placeholder string) graphql.ApplicationUpdateInput {
 	return graphql.ApplicationUpdateInput{
 		Name:                placeholder,
 		Description:         &placeholder,
-		HealthCheckURL:      &placeholder,
+		HealthCheckURL:      ptr.String("kyma-project.io"),
```

Comment:
What about the protocol part of the URL, it is omitted here on purpose?

Reply:
No